byron/chain/executable-spec/test/Cardano/Spec/Chain/STS/Properties.hs

```diff
@@ -1,24 +1,31 @@
+{-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
+
 module Cardano.Spec.Chain.STS.Properties where
 
-import           Control.Lens ((^.), (^..))
+import           Control.Arrow ((***))
+import           Control.Lens (view, (&), (^.), (^..), _1, _5)
 import           Data.Foldable (traverse_)
 import           Data.List.Ordered (nubSortBy)
 import           Data.Ord (Down (Down), comparing)
-import           Hedgehog (MonadTest, Property, assert, failure, forAll, property, withTests, (===))
+
+import           Hedgehog (MonadTest, Property, assert, cover, failure, forAll, property, withTests,
+                           (===))
 
 import           Control.State.Transition
 import           Control.State.Transition.Generator (TraceLength (Maximum), classifyTraceLength,
-                     traceSigGen)
+                                                     traceSigGen)
```

Comment:
Check that you have the `master` version of `stylysh-haskell` installed, and that it is using our configuration.

Reply:
oops, done